samples/besu/simpleasset/contracts/manageAssetAny.sol

```diff
@@ -0,0 +1,245 @@
+// SPDX-License-Identifier: Apache-2.0
```

Comment:
This file should go in `core/networks/besu/contracts/interop` folder.
Any contract that is core for weaver and not application dependent goes there.
Sample contains only sample application to demonstrate how to use core modules to enable HTLC/interoperation.

So then during installation/migration of contracts, you should copy it here and then migrate all of them together (that's how we deploy contracts in besu, since solidity files are in different folders, if you have better idea you can propose it.)

Reply:
this file should be deleted from here, and probably add this path in `.gitignore` of `simpleasset`, so that it doesn't get checked in by mistake.